New tests for abspos bugs in Chrome/Webkit

faceless2 opened the pull request:
Two abspos tests showing up positioning bugs in Chrome/Webkit

Commit:
New tests for abspos bugs in Chrome/Webkit

frivoal commented:
* -009 here seems redundant with css/css-text/line-breaking/line-breaking-016.html and css/css-text/line-breaking/line-breaking-018.html
* I don't think we have an earlier test for what -010 covers, but why only rel="help" it to css2, rather than to https://drafts.csswg.org/css-text-3/#text-encoding

Commit:
Delete 009, renamed 010 to 009, added new "help"

faceless2 commented:
Thanks for looking at these Florian - good spot on the line-breaking tests, yes it's exactly the same issue. I've added the help link as you suggested, deleted the old 009 test and renamed 010 to 009 to keep it sequential.

dbaron commented:
closing and reopening to trigger rerunning of tests

dbaron commented:
Seems like this needs to be merged/rebased with `master` in order to pass checks.
